docker/entrypoint.sh

```diff
@@ -246,7 +246,7 @@ case "$1" in
         list_directory "${ZEBRA_CACHED_STATE_DIR}"
         # Run both the fully synced RPC test and the subtree snapshot test, one test at a time.
         # Since these tests use the same cached state, a state problem in the first test can fail the second test.
-        run_cargo_test "${ENTRYPOINT_FEATURES}" "fully_synced_rpc_"
+        run_cargo_test "${ENTRYPOINT_FEATURES}" "--test-threads 1 fully_synced_rpc_"
```

Comment:
This is probably going to cause argument quoting issues, because `cargo test ... "--test-threads 1 fully_synced_rpc_"` is one argument, but we need `cargo test ... --test-threads 1 fully_synced_rpc_` (3 arguments).

If this test fails, I'd suggest adding the arguments inside the `run_cargo_test()` function. And if it doesn't fail, let's check the test logs to make sure it's actually running.

Here's an example of the difference:
```sh
$ cargo test -- --test-threads 1 test_name
    Finished test [unoptimized + debuginfo] target(s) in 0.00s
     Running unittests src/main.rs (target/debug/deps/empty_project-dd6fff5bb63559a4)

running 1 test
test test_name ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

$ cargo test -- "--test-threads 1 test_name"
    Finished test [unoptimized + debuginfo] target(s) in 0.00s
     Running unittests src/main.rs (target/debug/deps/empty_project-dd6fff5bb63559a4)
error: Unrecognized option: 'test-threads 1 test_name'
error: test failed, to rerun pass `--bin empty-project`
Exit 101
```

Reply:
Fixed with [fix: allow 3 arguments instead of 2 to handle fully_synced_rpc_ cor…](https://github.com/ZcashFoundation/zebra/pull/7660/commits/fed758bc44cb1e73620dc82b7c3536a8f61f8a72)